### PR TITLE
Update mu-features.yml

### PR DESCRIPTION
**Pull request by juanpedromoreno**
This PR updates the sample messages for the `Mu` landing page: http://higherkindness.io/mu-services/

### CODE REVIEW

File: _data/mu-features.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 content:
     - title: HTTP
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: REST based microservices, where server-side data are made available through representations of data in JSON.
```
Comment (raulraja) [inline _data/mu-features.yml]: Is this true though? There may be Http response types that are not represented in Json. For example an image service that serves binary data over http for the web

**Comment by juanpedromoreno** [comment]
@raulraja: Updated in https://github.com/higherkindness/higherkindness.github.io/pull/7/commits/3d61cab2821e4025b025b9dea0e93c1e55ba1c01 👍

**Comment by juanpedromoreno** [comment]
@raulraja ^^

File: _data/mu-features.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 content:
     - title: HTTP
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: REST based microservices, where server-side data are made available through representations of data, typically in JSON or XML format.
```
Comment (L-Lavigne) [inline _data/mu-features.yml]:
```suggestion
      description: REST-based typed microservices with auto-derived client and server implementations.
```

Comment (L-Lavigne) [reply _data/mu-features.yml]: Suggested wording tries to describe more unique features of the library (I only know about the upcoming Scala implementation however, not sure if this applies to Kotlin.)

**Comment by juanpedromoreno** [comment]
Done @L-Lavigne

**Pull request merged**